src/irclog2html/irclog2html.py

```diff
@@ -373,7 +374,7 @@ class AbstractStyle(object):
     description = "Single-line description"
     charset = 'US-ASCII'
 
-    def __init__(self, outfile, colours=None):
+    def __init__(self, outfile, outfilename='', colours=None):
```

Comment:
I suggest 
```suggestion
    def __init__(self, outfile, *, outfilename='', colours=None):
```
so that we can be sure there's no code that's creating styles by invoking `style(outfile, colours)` as positional arguments that will now suddenly change meaning.

Reply:
Ah, this does mean that you also have to change all the calls to do `style(outfile, outfilename=outfilename, colours=colours)`.  The tests are currently failing because of this.